action/protocol/staking/contractstaking/cache.go

```diff
@@ -247,6 +230,65 @@ func (s *contractStakingCache) BucketTypeCount() uint64 {
 	return uint64(len(s.bucketTypeMap))
 }
 
+func (s *contractStakingCache) LoadFromDB(kvstore db.KVStore) error {
```

Comment:
`loadFromDB()`, this is internal func
also, do we have a test to test this func?